Guardian/API/APIClient.swift

```diff
@@ -55,4 +55,20 @@ struct APIClient: API {
     func device(forEnrollmentId id: String, token: String) -> DeviceAPI {
         return DeviceAPIClient(baseUrl: baseUrl, id: id, token: token)
     }
+
+    func device(forEnrollmentId enrollmentId: String, userId: String, signingKey: SigningKey) -> DeviceAPI {
+        let responseExpiration: TimeInterval = 60 * 2 // 2 hs
```

Comment:
Why not have a separate method that creates this token and use it in the existing method?

Reply:
Because it's not used anywhere else. See https://github.com/auth0/Guardian.swift/blob/master/Guardian/Authentication/Authentication.swift#L141 for a similar example.
Can change it if it's a blocker.

Reply:
Not a blocker! It was a smell to me sending the `signingKey` and creating the token there. But it's minor.